scripts/config.py

```diff
@@ -85,19 +85,35 @@ def random_string(len):
     arr = [system_random.choice(chars) for i in range(len)]
     return ''.join(arr)
 
-def get_duo_data():
+def get_mfa_type():
     while True:
-        duo_resp = prompt('Will you be using DUO for MFA: (y/N)', default='N')
-        if (duo_resp == 'y' or duo_resp == 'Y'):
-            host = prompt("DUO api host, e.g. api-XXXXXXXX.duosecurity.com")
-            ikey = prompt("DUO integration key")
-            skey = prompt("DUO secret key")
-            return {'api_host': host, 'ikey': ikey, 'skey': skey}
-        elif (duo_resp == 'n' or duo_resp == 'N'):
-            return None
+        mfa_resp = prompt('Will you be using MFA: (y/N)', default='N')
+        if (mfa_resp == 'y' or mfa_resp == 'Y'):
+            while True:
+                mfa_type = prompt('What MFA provider: (Duo/Okta/cancel)', default='cancel')
```

Comment:
prompt should be (duo/okta/Cancel) -- capital C implies it's the default choice.